overlord/state/notices.go

```diff
@@ -205,11 +205,20 @@ const (
 	// Recorded by "snap run" command when it is inhibited from running a
 	// a snap due an ongoing refresh.
 	SnapRunInhibitNotice NoticeType = "snap-run-inhibit"
+
+	// Recorded whenever a request prompt is created or resolved. The key for
+	// interfaces-requests-prompt notices is the request prompt ID.
+	RequestsPromptNotice NoticeType = "interfaces-requests-prompt"
+
+	// Recorded whenever a request rule is created, modified, deleted, or
+	// expired. The key for interfaces-requests-rule-update notices is the
+	// rule ID.
+	RequestsRuleUpdateNotice NoticeType = "interfaces-requests-rule-update"
```

Comment:
Hi @pedronis , @alexmurray suggested naming these something like "request-prompt" and "request-rule", how would you feel about this?

Also, request prompt notices are re-issued whenever some new permissions are satisfied, and also whenever a listener request is merged into the existing prompt, so I'm not sure the distinction between "...-prompt" and "...-rule-update" makes the most sense anymore. Do you think these should both be "...-update", or should neither have the "...-update" suffix?

Reply:
I really don't see how we can get away without mentioning "interfaces" in those names without confusion,  I think the names are ok as their are, rules are long lived, so the "-update" bit there seems to make sense to me

Reply:
My comment was more around the fact that the other instances here are all named as `FooBarNotice NoticeType = "foo-bar-notice"` - but these all have an unexpected interfaces- prefix - so then either this interface- prefix is dropped or the variables should have an `Interfaces` prefix added to match. That's all. I don't really care what they are called, it was the inconsistency that was bugging me.

Reply:
Ahh, I see what you mean, thanks for pointing that out. I'll add the `Interface` prefix to the variable names as well.